### ISSUE TITLE
Missing documentation

### ISSUE THREAD

**Issue by monkey-knuckles** [koalaman/shellcheck#2993]
#### For bugs
- Rule Id (if any, e.g. SC1009): 
- My shellcheck version 0.10.0 (`shellcheck --version` or "online"): 
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC009)

#### For new checks and feature suggestions
- [ ] https://www.shellcheck.net/ (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh

echo 'Nothing so needs reforming as other people's habits.'

```

#### Here's what shellcheck currently says:

The mentioned syntax error was in this simple command.

#### Here's what I wanted or expected to see:

Either a page to explain error SC1009 or no report of that error

**Comment by brother** [comment]
If you use the online checker the error code links to the wiki page: https://github.com/koalaman/shellcheck/wiki/SC1009
![bild](https://github.com/koalaman/shellcheck/assets/243351/12e5972e-4de6-4d32-bf3d-4464158d5cc5)
